### DISCUSSION TITLE
Idea/Request: an option for 'close pop-up after action'

### DISCUSSION THREAD

**Discussion by PineappleEmperor** [Clooos/Bubble-Card#341] [Feature requests]
An idea/expansion for closing the pop-ups: The way I use the majority of my pop-ups is to provide a variety of options, that usually results in a single selection. I use the time based close or manual close but it'd be great to have the option to auto-close based on AN action being fired.

**Comment by Clooos** [comment]
Hi and thanks for the idea! I've just added this feature and it will be available in the next release 🍻

<img width="711" alt="image" src="https://github.com/Clooos/Bubble-Card/assets/36499953/c7942d94-902c-471c-93ee-98e2e0ff4072">

**Reply by haitham506** [reply]
Hello, in version 1.5.3 and before if you click outside of the Pop-up it closes it, it doesn't work like this in v1.6.*

**Reply by Clooos** [reply]
Hi, this should be fixed in the next release.

**Comment by Clooos** [comment]
This is now available in the new 1.6.3 version.

https://github.com/Clooos/Bubble-Card/releases/tag/v1.6.3

**Reply by PineappleEmperor** [reply]
This is so awesome! Thank you so much! I'm glad you liked the idea so much! =)

**Reply by Clooos** [reply]
You're welcome!
